packages/block-editor/src/components/block-list/index.native.js

```diff
@@ -67,6 +68,14 @@ export class BlockList extends Component {
 		);
 	}
 
+	showGroupAppender() {
```

Comment:
could we rename this to signal about the boolean return value? like `shouldShowGroupAppender `?

Reply:
Right, it point more clearly about returned value